.changeset/honest-timers-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@qwik-ui/headless': patch
+---
+
+bind syntax Sig suffix removed, new bind:selectedIndex prop, bug fixes
```

Comment:
I would separate type of changes into different commits, where each commit has it's own changeset

For example, one could be:

FEAT: (select) new `bind:selectedIndex` prop
Removed bind

Another could be

FIX: (select) now it's rendering correctly (or something like that)